guides/common/modules/con_working-with-host-groups.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Working_with_Host_Groups_{context}"]
+= Working with Host Groups
+
+A host group acts as a template for common host settings.
+Instead of adding host settings individually for each host, use host groups to define common settings once and apply them across multiple hosts.
```

Comment:
I'm not sure about the "new", though. If I reassign a registered host to another host group, the settings of the new host group will be applied and used e.g. during re-provisioning, right? @jeremylenz

Reply:
As I understand it, inherited attributes are applied on every host update, but _only_ for those attributes not already set on the host. To achieve this the host form sends all existing host attributes on any update, as well as any you're changing. Any attributes not set this way will then be inherited from the new host group. cc @ShimShtein to double check me

Reply:
> applied and used e.g. during re-provisioning

I'm not entirely sure what re-provisioning means here exactly?

Reply:
@Lennonka I agree with the tweaks for consistent terminology, so I applied that part of your suggestion: https://github.com/theforeman/foreman-documentation/pull/2685/commits/906b8533990c8de09c587591095f8c297834eec3
But I'd be super careful about using words like "new host" etc. at this point. Because the way I understand the behavior, the changes apply differently to newly created hosts and to existing hosts (it's part of why it's more of a "host template" than "host group").

My plan is to explain this in the procedures for adding a new host, changing an existing host's host group, adding an existing host to a host group etc. (WIP PR https://github.com/theforeman/foreman-documentation/pull/2689). Explaining all of this here in the conceptual parts of the assembly would just give users a headache.

Can we agree on this @Lennonka? Once the procedure modules are updated too, we can revisit this issue and see if it works like this or if some updates to the conceptual parts of the assembly are perhaps needed too.

Reply:
Totally! Thank you